src/course-outline/status-bar/StatusBar.jsx

```diff
@@ -11,6 +11,7 @@ const StatusBar = ({
   statusBarData,
   isLoading,
   courseId,
+  learnMoreVisibilityUrl,
```

Comment:
ref: remove as described in the comment above

Reply:
Done